commands/pipelinecmd.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt/internal/pipeline"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+// GetPipelineCommand returns a command that implements "kpt pipeline <run|...>" subcommands.
+func GetPipelineCommand(name string) *cobra.Command {
```

Comment:
I think we should try to stick with the `runner`-pattern for the cobra commands: https://github.com/GoogleContainerTools/kpt/blob/655b215629ac693b736154a453dbf5a20d08e9a4/internal/cmdget/cmdget.go#L32

Reply:
Ack. I started without Runner with a plan to add it when I need it, but didn't encounter any need. 
I think I will probably add it to wrap this command to include some auxiliary stuff such as logger, commandline options. Curious where else did you find runner useful ?

Reply:
Moved the `run` subcommand to runner pattern as you suggested.